### PR TITLE
adding targetURL when using `status` action

### LINKED ISSUE #117: Add option to provide a targetURL when using `status` action
I thought it will be useful to provide a details link in the rule when using `status` action, probably when trying to provide a link to documentation related to that validation/use

### PR DESCRIPTION
**Pull request by gagoar**
**Issue Reference**: fixes #117 

## Description

This Pr allows you to provide a targetURL that it will be the 

## Motivation and Context

I thought it will be pretty useful to offer a link on the details that takes you to documentation about that validation or some useful link

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,6 +85,7 @@ Every rule can be written in JSON with the following key-value pairs:
...
 | `users`           |       `string[]`       |    No    | GitHub user handles (or emails) on which the rule will take action. It will not be used when `action` is set to `comment` and `customMessage` field is present                                                                                           |
 | `teams`           |       `string[]`       |    No    | GitHub teams on which the rule will take action. It will not be used when `action` is set to `comment` and `customMessage` field is present                                                                                                              |
+| `targetURL`       |        `string`        |    No    | It will be used when `action` is set to `status` as the link under [Details](https://github.blog/2012-09-04-commit-status-api/)                                                                                                                          |
```
Comment (cyamonide) [inline README.md:88]:
```suggestion
| `targetURL`       |        `string`        |    No    | When `action` set to `status`, link to which the [Details](https://github.blog/2012-09-04-commit-status-api/) link will point                                                                                                                          |
```

File: herald_rules/validate_description_is_present.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
   "name": "The template has a Description",
+  "targetURL": "https://github.com/gagoar/use-herald-action/wiki/Pr-Validations#validate-description-title-is-present",
```
Comment (cyamonide) [inline herald_rules/validate_description_is_present.json:3]:
```suggestion
  "targetURL": "https://github.com/gagoar/use-herald-action/wiki/PR-Validations#validate-description-title-is-present",
```
I have already changed the respective wiki page

File: herald_rules/validate_issue_is_present.json [hunk 3]
```diff
@@ -1,6 +1,7 @@
 {
   "name": "The template has an issue attached",
-  "description": "Check if an issue present on PR",
+  "description": "Check if an issue is present on PR",
+  "targetURL": "https://github.com/gagoar/use-herald-action/wiki/Pr-Validations#validate-if-the-issue-is-present-in-the-prs-description",
```
Comment (gagoar) [inline herald_rules/validate_issue_is_present.json:4]:
```suggestion
  "targetURL": "https://github.com/gagoar/use-herald-action/wiki/PR-Validations#validate-if-the-issue-is-present-in-the-prs-description",
```

**Review by cyamonide — APPROVED** [review]
LGTM

**Pull request merged**